scripts/clone-react-dom-test-suite/utils.js

```diff
@@ -0,0 +1,18 @@
+const { exec } = require('child_process');
+const { transform } = require('babel-core');
+
+module.exports.clone = function(url, dirname) {
+  return new Promise((resolve, reject) => {
+    exec(`git clone --depth 1 ${url} ${dirname}`, error => {
```

Comment:
i'd use the `execa` pacakge and `fs-extra` for promise apis (easier then rolling our own every time)

Reply:
I usually avoid third party if requirments are trivial :)

Reply:
Why maintain the code tho? That's the advantage to the big ecosystem :)

Reply:
True :)